server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -438,10 +460,15 @@ private File compileFile(int index, File extDir, File src, Map<String, Object> m
         // Add the other extensions include folders
         for (File otherExtDir : this.getExtensionFolders()) {
             File otherIncludeDir = new File(otherExtDir, "include");
-            if (extIncludeDir.equals(otherIncludeDir))
+            if (extIncludeDir.equals(otherIncludeDir)) // don't add the current extension's include directory
                 continue;
             includes.add( ExtenderUtil.getRelativePath(jobDirectory, otherIncludeDir ) );
         }
+        return includes;
+    }
+
+    private File addCompileFileCpp_Internal(int index, File extDir, File src, Map<String, Object> manifestContext, String cmd, List<String> commands) throws IOException, InterruptedException, ExtenderException {
```

Comment:
The old name "compileFile" was now a misnomer, since it only sets up the command.